tsconfig.json

```diff
@@ -1,8 +1,12 @@
 {
 	"extends": "aegir/src/config/tsconfig.aegir.json",
 	"compilerOptions": {
```

Comment:
might want to make the ts rules a bit more strict, still allows quiet some potential errors from what I can tell, or perhaps I am wrong. You tell me.